torch/csrc/jit/codegen/cuda/nvfuser.cmake

```diff
@@ -9,13 +9,16 @@ endif()
 # The list of NVFUSER runtime files
 list(APPEND NVFUSER_RUNTIME_FILES
   ${TORCH_SRC_DIR}/csrc/jit/codegen/cuda/runtime/array.cu
+  ${TORCH_SRC_DIR}/csrc/jit/codegen/cuda/runtime/array_rocm.cu
```

Comment:
nitpick. Can we guard rocm only files under `if(USE_ROCM)`?

Reply:
What about the [bzl file](https://github.com/pytorch/pytorch/blob/6bbe7a79fbfd45b8b0b0ecfa7cdabc0822558e19/build_variables.bzl)?  

The overhead of keeping these four rocm-specific files in the NVFUSER_RUNTIME_FILES cmake var is minimal; four extra files to stringify at build time.  I know it is a nit, but I'd rather avoid yet another commit to this PR since we're in a  good state at the moment.

Reply:
I'll accept & land this, and @jeffdaily can you open another PR to address Jie's comments once this lands?

Reply:
See: https://github.com/pytorch/pytorch/pull/84312.